bots/tests-scan

```diff
@@ -463,9 +456,10 @@ def cockpit_tasks(api, update, contexts, repo, pull_data, pull_number, sha, amqp
             return False
 
         if trigger_externals():
-            for proj_repo, proj_contexts in itertools.chain(
-                    EXTERNAL_PROJECTS.items(), REDHAT_EXTERNAL_PROJECTS.items()):
-                for context in proj_contexts:
+            external_repos = list(REPO_BRANCH_CONTEXT.keys())
+            external_repos.remove("cockpit-project/cockpit")
+            for proj_repo in external_repos:
+                for context in REPO_BRANCH_CONTEXT[proj_repo]["master"]:
```

Comment:
Nitpick: don't make a copy, and just check/`continue` in the loop if the repo is cockpit.

Reply:
`.keys()` should not be necessary here (but this is unimportant).